### ISSUE TITLE
Feature: Create open-wa binary for zero-install api

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#604]
example:
```bash
> npx @open-wa/wa-automate --api --use-chrome --port 8002
```

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.99](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.98...1.9.99) (2020-07-16)

- build(deps-dev): bump marked from 1.1.0 to 1.1.1 [`#649`](https://github.com/open-wa/wa-automate-nodejs/pull/649)
- build(deps-dev): bump @types/node from 14.0.22 to 14.0.23 [`#650`](https://github.com/open-wa/wa-automate-nodejs/pull/650)
- build(deps): bump puppeteer from 5.0.0 to 5.1.0 [`#643`](https://github.com/open-wa/wa-automate-nodejs/pull/643)
- build(deps-dev): bump @types/node from 14.0.20 to 14.0.22 [`#644`](https://github.com/open-wa/wa-automate-nodejs/pull/644)
- Completed common type alias migration #583 [`9b7673b`](https://github.com/open-wa/wa-automate-nodejs/commit/9b7673be2380afa32057b9ac8faa688bbbf91d6f)
- Feature: `inDocker` will try to find most configs from process.env #603 [`fadf126`](https://github.com/open-wa/wa-automate-nodejs/commit/fadf1261b1efa226f26b9938dd0674fce87e805c)
- Release 1.9.99 [`2e8b768`](https://github.com/open-wa/wa-automate-nodejs/commit/2e8b768924c2c5901be0240e6b6bfed9ec7dcce1)
- Feature: Implemented simple server binary `npx open-wa -p 8080` #604 [`6f4f3f1`](https://github.com/open-wa/wa-automate-nodejs/commit/6f4f3f15009116e4fda527621578245cb4eefe40)
- implemented docwatch and new docs theme [`3e4b3f2`](https://github.com/open-wa/wa-automate-nodejs/commit/3e4b3f269a591820b1748ebc638439400a4e23bf)
- Added better error handling on sendText #645 [`53660d5`](https://github.com/open-wa/wa-automate-nodejs/commit/53660d55d7cd0aa806f445f419115a86955386fe)
- BREAKING CHANGE: client.page is now deprecated and private. access via `client.getPage()` [`2acfd6b`](https://github.com/open-wa/wa-automate-nodejs/commit/2acfd6b923f0713e5d55ec50b8e382540e99f341)
- chore: update supported version in readme [`ccbf1d2`](https://github.com/open-wa/wa-automate-nodejs/commit/ccbf1d2bdc7745f5cffa2031ad5dcb59406e4281)
- fix demos [`a5da92c`](https://github.com/open-wa/wa-automate-nodejs/commit/a5da92c01fec8e76710375dd5d455c86a82d2e93)
- Feature: `sessionDataPath` now can point to specific session.data.json file instead of just directory [`1d8d31e`](https://github.com/open-wa/wa-automate-nodejs/commit/1d8d31e941205e10f070700a4df4eb677530b794)
- `killProcessOnBrowserClose` default to false #647 [`96b7d4e`](https://github.com/open-wa/wa-automate-nodejs/commit/96b7d4e92056e9f75dc0fe65497c80105ba3ff7c)
- chore: Update default UA [`2bab677`](https://github.com/open-wa/wa-automate-nodejs/commit/2bab677b85b980b4d7422fada896e7bdcd02447f)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.100](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.100-0...1.9.100) (2020-07-17)

- Release 1.9.100 [`b4f8175`](https://github.com/open-wa/wa-automate-nodejs/commit/b4f8175ac02aff59d2c8877252e691a75ed20f0e)
- feature: npx integration fixes (#658) #604 [`47596f9`](https://github.com/open-wa/wa-automate-nodejs/commit/47596f99b6667e77119823b6c0c1fe4347e020fb)
- update wapi.js getAllMessagesInChat [`6515bbb`](https://github.com/open-wa/wa-automate-nodejs/commit/6515bbb4e42005dd8f291857b58b761978f48826)
- serialize message output on clean: false in `getAllMessagesInChat` [`4f917a0`](https://github.com/open-wa/wa-automate-nodejs/commit/4f917a0499057ace001e5479329b25030c641e3c)

**Comment by smashah** [comment]
You can now create an API of your WhatsApp number simply via the terminal:

```bash
> npx @open-wa/wa-automate 
```
